test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetPackCommandTest.cs

```diff
@@ -1726,9 +1726,9 @@ public void PackCommand_ReferencedProjectWithNuspecFileWithMsbuild14()
                 Assert.Equal(
                     new string[]
                     {
-                        Path.Combine("lib", "net40", "proj1.dll"),
-                        Path.Combine("lib", "net40", "proj3.dll"),
-                        Path.Combine("lib", "net40", "proj7.dll")
+                        Path.Combine("lib", "net48", "proj1.dll"),
```

Comment:
how did this change to net48?
I'm not sure which particular change caused it?

Reply:
Since we are moving to VSEngSS-MicroBuild2022-1ES pool, we need to get rid of net45. Please see [Visual Studio 2022 Support for .NET Development](https://docs.microsoft.com/en-us/visualstudio/releases/2022/compatibility#-visual-studio-2022-support-for-net-development) for more details.

Reply:
My question is more about where's the code that made this go from net40 to net48. I had a hard time finding it

Reply:
You're right! Those `net40` won't failed the tests as there is no need to create projects.
Revert changes in this file. Thanks!